recommenders/models/ncf/ncf_singlenode.py

```diff
@@ -4,13 +4,13 @@
 import os
 import numpy as np
 import tensorflow as tf
+import tf_slim as slim
```

Comment:
I'm curious, what is the difference between the layers in tf_slim and the layers of TF or Keras?

Reply:
It looks like they moved most of the layers from `tf.contrib` to `tf_slim`. Keras API differs from TF1 API in many ways, including how to define layers, see https://www.tensorflow.org/guide/migrate/tf1_vs_tf2.